libraries/ts-command-line/src/providers/CommandLineParameterProvider.ts

```diff
@@ -45,11 +64,16 @@ export interface ICommandLineParserData {
  * @public
  */
 export abstract class CommandLineParameterProvider {
+  private static readonly _scopeGroupName: string = 'scope';
+  private static readonly _songNameGroupName: string = 'longName';
+  private static readonly _scopedLongNameRegex: RegExp =
+    /^--((?<scope>[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*):)?(?<longName>[a-z0-9]+((-[a-z0-9]+)+)?)$/;
```

Comment:
Since `--long-name` enforces kebab case, it seems inconsistent to allow `--PascalCase:long-name`.  Furthermore, for plugin names we really do prefer long descriptive names like `Storybook2GalleryPlugin`, whereas CLI parameters have the exact opposite requirement of being short and easy to type `--storybook2:long-name`.  Maybe the Heft plugin manifest should require a short kebab name (`storybook2`) for this purpose.

Reply:
Yeah agreed on that.